frontend/src/utils/role.ts

```diff
@@ -16,7 +16,9 @@ export type WorkspacePermissionType =
   | "bb.permission.workspace.manage-general"
   | "bb.permission.workspace.manage-sensitive-data"
   // Can execute admininstrive queries such as "SHOW PROCESSLIST"
-  | "bb.permission.workspace.admin-sql-editor";
+  | "bb.permission.workspace.admin-sql-editor"
+  // Can create the service account and get the token
+  | "bb.permission.workspace.manage-service-account";
```

Comment:
Introduce the `bb.permission.workspace.manage-service-account` permission. Should we grant this permission to both `DBA` and `Owner`, or only to the `Owner`? PTAL @tianzhou @d-bytebase

Reply:
Should be owner Only and I feel we can reuse bb.permission.workspace.manage-member

Reply:
Good for me, updated